### PR TITLE
[REVIEW] Update docs and AIP instructions

### PR DESCRIPTION
**Pull request by zronaghi**
Update README and rename mlflow_projects to mlflow to match other folders
Add GCP AI Platform instructions with updated RAPIDS env

### CODE REVIEW

**Comment by zronaghi** [comment]
@Nanthini10 added the updated Azure example

File: azure/notebooks/Train-RAPIDS.ipynb [hunk 1]
```diff
@@ -280,11 +271,18 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "#rapids-cloud-ml image is available in Docker Hub\n",
-    "image_name = 'rapidsai/rapidsai-cloud-ml:0.18-cuda11.0-base-ubuntu18.04-py3.8'\n",
+    "from azureml.core import Environment\n",
+    "\n",
+    "#environment file\n",
+    "environment_file = \"Dockerfile\"\n",
```
Comment (Nanthini10) [inline azure/notebooks/Train-RAPIDS.ipynb:277]: What does this Dockerfie correspond to? Is there supposed to be a dockerfile included in the directory?

Comment (zronaghi) [reply azure/notebooks/Train-RAPIDS.ipynb:277]: Yup created one so you can modify for your example, just pushed that as well

**Review by drobison00 — APPROVED** [review]
Looks good.

File: azure/notebooks/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FROM rapidsai/rapidsai-core:0.18-cuda11.0-base-ubuntu18.04-py3.8
+RUN apt-get update && \
+apt-get install -y fuse && \
+source activate rapids && \
+pip install azureml-mlflow && \
+pip install azureml-dataprep
```
Comment (Nanthini10) [inline azure/notebooks/Dockerfile:6]: Needs to include `pip install dask-ml`

**Review by Nanthini10 — CHANGES_REQUESTED** [review]
Looks good to me. I was able to run it. Just missing the `dask-ml` installation

File: azure/notebooks/Dockerfile (hunk 2 above)
Comment (zronaghi) [reply azure/notebooks/Dockerfile]: Awesome, added `dask-ml` as well!

**Pull request merged**